.pylintrc

```diff
@@ -2,7 +2,6 @@
 disable=bad-continuation,         ######## Disabled because of conflicts with black ########
         consider-using-enumerate, ######## Temporary disabled ########
         import-error,
-        too-many-statements,
```

Comment:
This looks like the _enabling_ of a diagnostics.

Reply:
The commit message should also be more verbose, not many know what `too-many-statements` is.

Proposal:
```
Enabling pylint diagnostics `too-many-statements`
```

Reply:
done, but it is 'Dissabling'

Reply:
I'm still confused. You are removing a disable-statement, and thus enabling it, no?

Reply:
My mistake.

Reply:
Well, the problem was that this enables diagnostics, but disables too many statements :)

Reply:
I've renamed the commit message.

Reply:
Looks good now.

Reply:
PR title is still weird.

Reply:
renamed it to: Enable pylint diagnostics `too-many-statements`

Reply:
Need your approval @nedrebo  to merge.